users.tf

```diff
@@ -23,6 +23,17 @@ data "aws_iam_policy_document" "ci_user" {
       "*",
     ]
   }
+  statement {
+    actions = [
+      "s3:ListBucket",
+      "s3:PutObject"
+    ]
+
+    resources = [
+      aws_s3_bucket.assets.arn,
+      "${aws_s3_bucket.assets.arn}/*"
```

Comment:
use format instead of string interpolation for consistency

Reply:
Also, we use * but this is the CI user... as an optimization we could use the name of the bucket (ni-qa in this case) as a more restrictive version of this.